server/fleet/hosts.go

```diff
@@ -216,17 +216,32 @@ func (h *Host) FleetPlatform() string {
 	return PlatformFromHost(h.Platform)
 }
 
+// HostLinuxOSs are the possible linux values for Host.Platform.
+// TODO(lucas): Osquery could also return `gentoo`, `posix`, `freebsd`.
+var HostLinuxOSs = []string{
+	"linux", "ubuntu", "debian", "rhel", "centos",
```

Comment:
Let's add these as suggested in https://github.com/fleetdm/fleet/issues/3211#issuecomment-986503790

```suggestion
	"linux", "ubuntu", "debian", "rhel", "centos", "sles", "kali",
```

Reply:
Sure, have we confirmed osquery can generate those?  (how do we know  possible values)

According to osquery source code there's also `gentoo`, `freebsd` and `posix` (when it cannot determine linux flavor).

Reply:
/cc @sharvilshah

Reply:
I confirmed `sles` and `gentoo` by installing osquery in Docker containers.

Reply:
OK, @zwass I guess it doesn't hurt to at least add `sles`, `kali` and `gentoo` to the list for now, right?

Reply:
Confirmed `kali` as well. I don't think it hurts to add any of them.

Reply:
Thank you for verifying! They have been added.